lib/Widgets/home/teleblitz.dart

```diff
@@ -74,24 +90,170 @@ class Teleblitz {
   }
 
   Widget teleblitz() {
-    return MoreaShadowContainer(
-      child: Padding(
-        padding: const EdgeInsets.all(15),
-        child: Column(
-          children: <Widget>[
-            info.getTitel(),
-            info.getDatum(),
-            info.getAntreten(),
-            info.getAbtreten(),
-            info.getMitnehmen(),
-            info.getBemerkung(),
-            info.getSender(),
-          ],
+    if (moreaFire.getPos != "Leiter") {
```

Comment:
Was ich vermeiden möchte ist ein Source-code der 700+ Zeilen hat. Ich finde dies unübersichtlich.
Aus diesem Grund habe ich angefangen das UI und die Logik zu trennen und zu seperieren. Vieleicht müssten wir und mal besprechen was in werches File kommt. 
Mein Gedanke: 
Die Klasse teleblitzAnmeldungen hat zwei Funktionen:
 `Stream<List<String>> getTNAngemolden(String eventID);`
und
  `Stream<List<String>> getTNAbgemolden(String eventID);`
Die übergeben dem Widget chunt und chuntnoed diesen Stream, der wiederum ein ListView generiert. Kannst du nicht einfach dies umschreiben und die Widgets aus Werchunt verwenden?

Reply:
aber genau das habe ich ja getan...
WerChunnt macht den stream und teleblitz zeigt ihn als listview an.

Reply:
Ja egal, wenn wir am gleichen arbeiten, müsste wir uns in Zukunft besser absprechen.

Reply:
Ja das denke ich auch. Ich denke wir versuchen so schnell wie möglich eine stabile v.1 herauszubringen und dann können wir ja an der Übersichtlichkeit und innere Einigkeit des Codes arbeiten.